Tests/CommandsTests/APIDiffTests.swift

```diff
@@ -402,18 +396,22 @@ class APIDiffTestCase: CommandsBuildProviderTestCase {
             let repo = GitRepository(path: packageRoot)
             let revision = try repo.resolveRevision(identifier: "1.2.3")
 
-            await XCTAssertThrowsCommandExecutionError(
-                try await execute(["diagnose-api-breaking-changes", "1.2.3", "--baseline-dir", baselineDir.pathString], packagePath: packageRoot)
+            await expectThrowsCommandExecutionError(
+                try await execute(["diagnose-api-breaking-changes", "1.2.3", "--baseline-dir", baselineDir.pathString], packagePath: packageRoot, buildSystem: buildSystem)
             ) { error in
-                XCTAssertMatch(error.stdout, .contains("1 breaking change detected in Foo"))
-                XCTAssertMatch(error.stdout, .contains("💔 API breakage: func foo() has been removed"))
-                XCTAssertFileExists(baselineDir.appending(components: revision.identifier, "Foo.json"))
+                expectMatch(error.stdout, .contains("1 breaking change detected in Foo"))
+                expectMatch(error.stdout, .contains("💔 API breakage: func foo() has been removed"))
+                if buildSystem == .swiftbuild {
```

Comment:
suggestion (non-blocking): can we only put the delta in the body of the conditional to make it the difference more explicit?

e.g.:

```swift
if buildSystem == .swiftbuild {
   let expectedFileName = "Foo"
else {
   let expectedFileName = "Foo.json"
}

#expect(localFileSystem.exists(baselineDir.appending(components: revision.identifier, expectedFileName)))
```


This comment applies to all other similar code changes